README.md

```diff
@@ -4,8 +4,13 @@
 [![](https://img.shields.io/librariesio/dependents/nuget/FluentAssertions.svg?label=dependent%20libraries)](https://libraries.io/nuget/FluentAssertions)
 ![](https://img.shields.io/badge/release%20strategy-githubflow-orange.svg)
 
-# What is this and why do I need this?
-See https://www.fluentassertions.com for background information, usage documentation, an extensibility guide, support information and more tips & tricks.
+# About this project
+A very extensive set of extension methods that allow you to more naturally specify the expected outcome of a TDD or BDD-style unit tests. Targets .NET Framework 4.5 and 4.7, as well as .NET Core 2.0, .NET Core 2.1, .NET Core 3.0, .NET Standard 1.3, 1.6 2.0 and 2.1.
```

Comment:
note: We do don't target all those TFM in develop

Reply:
That's what you get by blindly upmerging things. Apparently this wasn't even mentioned in the read-me on `develop`.